.gitignore

```diff
@@ -31,4 +31,3 @@ build/
 
 ### VS Code ###
 .vscode/
-src/main/resources/application.properties
```

Comment:
Why did you remove this from the gitignore?

Reply:
Originally it was added to store and use an api key but not anymore